homeassistant/components/tibber/sensor.py

```diff
@@ -193,6 +193,33 @@
     ),
 )
 
+SENSORS: tuple[SensorEntityDescription, ...] = (
+    SensorEntityDescription(
+        key="month_cost",
+        name="Monthly cost",
+        device_class=SensorDeviceClass.MONETARY,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    SensorEntityDescription(
+        key="peak_hour",
+        name="Month peak hour consumption",
+        device_class=SensorDeviceClass.ENERGY,
+        native_unit_of_measurement=ENERGY_KILO_WATT_HOUR,
+    ),
+    SensorEntityDescription(
+        key="peak_hour_time",
```

Comment:
Does this return a timezone aware datetime object from the library?

Reply:
Yes, in local time